docs/serving/samples/hello-world/helloworld-nodejs/index.js

```diff
@@ -1,14 +1,14 @@
-const express = require('express');
+const express = require("express");
 const app = express();
 
-app.get('/', (req, res) => {
-  console.log('Hello world received a request.');
+app.get("/", (req, res) => {
+  console.log("Hello world received a request.");
 
-  const target = process.env.TARGET || 'World';
+  const target = process.env.TARGET || "World";
   res.send(`Hello ${target}!`);
 });
 
 const port = process.env.PORT || 8080;
 app.listen(port, () => {
-  console.log('Hello world listening on port', port);
+  console.log("Hello world listening on port", port);
 });
```

Comment:
All the changes in this file should be reverted.

Reply:
done